tests/test_classification.py

```diff
@@ -227,42 +227,42 @@ def check_label_index_order_with_pos_label_str(use_api=True):
         eq_(clf.label_list, correct_order)
         eq_(label_order_from_dict, correct_order)
         if len(unique_label_list) != 2:
-            eq_(clf.pos_label_str, None)
+            eq_(clf.pos_label, None)
         else:
-            eq_(clf.pos_label_str, pos_label_str)
+            eq_(clf.pos_label, pos_label)
 
 
-def test_label_index_order_with_pos_label_str():
-    yield check_label_index_order_with_pos_label_str, True
-    yield check_label_index_order_with_pos_label_str, False
+def test_label_index_order_with_pos_label():
+    yield check_label_index_order_with_pos_label, True
+    yield check_label_index_order_with_pos_label, False
 
 
-def check_binary_predictions_for_pos_label_str(label_list,
-                                               use_pos_label_str=True,
-                                               probability=True,
-                                               use_api=True):
+def check_binary_predictions_for_pos_label(label_list,
+                                           use_pos_label=True,
+                                           probability=True,
+                                           use_api=True):
     """
     This tests whether binary predictions and probabilities
     obtained from SKLL (via the API as well as the config)
     match the predictions from scikit-learn on the same data
-    for when `pos_label_str` is set and also when it is not set.
+    for when `pos_label` is set and also when it is not set.
     """
 
     if label_list == ['B', 'C']:
-        pos_label_str, other_label_str = 'B', 'C'
+        pos_label, other_label_str = 'B', 'C'
```

Comment:
I know that this PR is supposed to change only `pos_label_str` to `pos_label` but in a couple of other places, I did see `neg_label_str` being changed to `neg_label`. Similarly, should `other_label_str` be changed too?

Reply:
I checked out the branch and grepped all the `_label_str` and found the following occurrences of `other_label_str`.
```
./tests/test_classification.py:252:        pos_label, other_label_str = 'B', 'C'
./tests/test_classification.py:254:        pos_label, other_label_str = 2, 1
./tests/test_classification.py:256:        pos_label, other_label_str = 'FRAG', 'NONE'
./tests/test_classification.py:263:        index_label_dict = {0: other_label_str, 1: pos_label}
./tests/test_classification.py:265:        sorted_label_list = sorted([pos_label, other_label_str])
```

Reply:
Sure, we can do that for consistency. Thanks guys!